fecgov/fec-cms
# Feature/5278 election year filters for election summaries

## Summary

- Resolves #5278 

Changing the election totals filters from `min_election_cycle` and `max_election_cycle` to `election_year` values

### Required reviewers
(Feel free to remove yourself)
- front-end for code/approach
- back-end for data?
- SME for accuracy

## Impacted areas of the application

Only the 'contributions over time' section for the [House](http://127.0.0.1:8000/data/elections/house/) and [Senate](http://127.0.0.1:8000/data/elections/senate/) elections pages

## Screenshots

No visible changes

## Related PRs

None

## How to test

- pull the branch
- `npm run build`
- `./manage.py runserver`
- Check the date range pulldowns in the 'contributions over time' section of the [House elections page](http://127.0.0.1:8000/data/elections/house/)
- Check the date range pulldowns in the 'contributions over time' section of the [Senate elections page](http://127.0.0.1:8000/data/elections/senate/)
